src/pages/rekey-reason/rekey-reason.scss

```diff
@@ -1,4 +1,67 @@
 page-rekey-reason {
+
+  .transferInfo {
+    color: map-get($colors-gds, gds-grey-1);
+    font-weight: initial;
+    letter-spacing: initial;
+  }
+
+  .staffNumber {
+    max-width: 40%;
+    width: 40%;
+    min-width: 40%;
+  }
+
+  .staffNumberRow {
+    height: 70px;
+    max-height: 70px;
+  }
+
+  ion-spinner {
+    margin-left: 6px;
+    * {
+      stroke: black;
+    }
+
+    &.spinner-circles circle,
+    &.spinner-bubbles circle,
+    &.spinner-dots circle {
+      fill: black;
+    }
+  }
+
+  tick-indicator {
+    margin-top: 12px;
+    margin-left: 5px;
+    div {
+      width: 40px;
+      height: 40px;
```

Comment:
This height will need to be `min-height` so that the div grows when large text is enabled